app/assets/stylesheets/thyme.scss

```diff
@@ -341,3 +418,234 @@ figure {
  {
    background-color: black;
  }
+
+
+
+/* annotation modal */
+#annotation-modal {
+  .modal-content {
+    width: 100%;
+  }
+  #annotation-modal-content {
+    width: 100%;
+  }
+  #preview-toggle {
+    input:checked + .slider {
+      background-color: #0D6EFD;
+    }
+    .switch {
+      vertical-align: middle;
+    }
+  }
+  #annotation-modal-preview {
+    height: 543px;
+    width: 35%;
+    word-wrap: break-word;
+    overflow-y: scroll;
+  }
+  #annotation_comment {
+    height: 150px;
+    resize: none;
+  }
+  #annotation_category_text {
+    width: 200px;
+  }
+  input[type=checkbox] {
+    width: 18px;
+    height: 18px;
+  }
+}
+
+#emergency-link {
+  text-align: center;
+}
+
+#markers {
+  position: relative;
+  top: -2px;
+  width: 0;
+  height: 0;
+  display: flex;
+  cursor: pointer;
+}
+
+#annotation-caption {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+#annotation-infobar {
+  height: 2em;
+  color: black;
+  overflow-y: scroll;
+  padding: 3px;
+  border-left: 1px solid darkgray;
+  border-right: 1px solid darkgray;
+  border-top: 1px solid darkgray;
+}
+
+#annotation-comment {
+  margin-left: 3px;
+  margin-right: 3px;
+  overflow-y: scroll;
+  height: 85%;
+}
+
+#annotation-buttons {
+  display: flex;
+  justify-content: space-between;
+  margin-left: 10%;
+  margin-right: 10%;
+  margin-bottom: 15px;
+  margin-top: auto;
+}
+
+
+/* A collection of all colors.
+Be aware that color changes must also be written in the
+AnnotationHelper! */
+$annotation_colors: (
+  annotation_color1: #DB2828,
+  annotation_color2: #F2711C,
+  annotation_color3: #FBBD08,
+  annotation_color4: #B5CC18,
+  annotation_color5: #21BA45,
+  annotation_color6: #00B5AD,
+  annotation_color7: #2185D0,
+  annotation_color8: #6435C9,
+  annotation_color9: #A333C8,
+  annotation_color10: #E03997,
+  annotation_color11: #d05d41,
+  annotation_color12: #924129,
+  annotation_color13: #444444,
+  annotation_color14: #999999,
+  annotation_color15: #eeeeee,
+);
+
+* {
+  box-sizing: border-box;
+}
+
```

Comment:
Isn't this the default? Even if it isn't, please don't use `*`, as this will refer to every HTML element on every possible MaMpf page due to the bundling.

Reply:
Actually, removing these lines doesn't change anything as far as I see it (I thought, we needed it for a better behaviour of the color boxes, but eventually this is not the case).